### DISCUSSION TITLE
ae2022版本的，导出pag文件没有图片替换路径

### DISCUSSION THREAD

**Discussion by wuzhipeng555** [Tencent/libpag#2304] [General] [answered]
ae是2022版本的，原来有文字的文件导出后，文字路径（view）识别不出来。这是哪里出问题了。

**Comment by wuzhipeng555** [comment]
![去去凄凄切切凄凄切切](https://github.com/Tencent/libpag/assets/20653549/3e2051f1-389d-4971-95ae-ae9ecb2413b0)

**Comment by kevingpqi123** [comment, marked as answer]
https://pag.art/docs/ae-bmp-guide.html
pag 的官方文档可以阅读下
